yang/frr-bgp.yang

```diff
@@ -23,6 +23,10 @@ module frr-bgp {
     prefix frr-bt;
   }
 
+  import frr-pim-rp {
```

Comment:
It looks a little bit weird to include PIM modules here.
Could we move `ipv4/ipv6/ip-multicast-group-address-prefix` definitions to some common model like `frr-route-types`?